readme.txt

```diff
@@ -157,7 +157,7 @@ Enhancements:
 * Adds a notification when a post is removed.
 * Overhauls the Content Types section under SEO -> Search Appearance by sectioning the post types and allowing users to collapse them. This is especially handy when you have a lot of custom post types.
 * Updates the 'snippet variables tab' of the Help Center to have the new names.
-* Adds recommended snippet variables for templates depending on the context.
+* Adds recommended snippet variables for templates depending on the context. The `wpseo_recommended_replace_vars` filter is added, which gives the possibility to add or alter the recommended snippet variables.
```

Comment:
This is a strange change, can you remove this?
The `wpseo_recommended_replace_vars` is already mentioned in an entry above.